zoo/meilisearch/migrations/0001_index_existing_models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import migrations
+import meilisearch
+from django.conf import settings
+
+
+def create_new_indexes(apps, schema_editor):
+    client = meilisearch.Client(
+        "http://meilisearch:7700", settings.MEILISEARCH_MASTER_KEY
+    )
+
+    client.create_index(uid="services", options={"name": "Service", "primaryKey": "id"})
+    client.create_index(
+        uid="dependencies", options={"name": "Dependency", "primaryKey": "id"}
+    )
+    client.create_index(uid="open-api", options={"name": "Schema", "primaryKey": "id"})
+
+
+def remove_indexes(apps, schema_editor):
+    client = meilisearch.Client(
+        "http://meilisearch:7700", settings.MEILISEARCH_MASTER_KEY
```

Comment:
We should have this configurable via env var, and we can default to `"http://meilisearch:7700"` in a single place

```suggestion
        settings.MEILISEARCH_ENDPOINT, settings.MEILISEARCH_MASTER_KEY
```

Reply:
sure, this is still WIP mr

Reply:
alrighty, converted to draft